### PR TITLE
Music album milestone 3 : add and list all genres

### PR DESCRIPTION
**Pull request by Nemwel-Boniface**
## Hello :wave: 
In this milestone I was able to:

- [x] Create a genre album
- [x] Save a genre album
- [x] List all available genre albums
- [x] Check and resolve all found offenses
- [x] Save to JSON files
- [x] Retrieve and display from JSON files

Thank you for reviewing my project :pray:

### CODE REVIEW

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hi @Nemwel-Boniface , 

Good job so far!

## Highlight!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

File: musicAlbum/music_controller.rb [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
+require_relative './handle_data'
+require 'json'
 
 class MusicController
   attr_accessor :musics
 
   def initialize
-    @musics = []
+    @musics = read_musics
   end
```
[the remark this answers is not in the document]
Comment (Nemwel-Boniface) [reply musicAlbum/music_controller.rb:10]: Thank you Levy for this suggestion, let me effect it immediately.

File: musicAlbum/genre_controller.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+require_relative './genre'
+require_relative './handle_data'
+require 'json'
+
+class GenreController
+  attr_accessor :musics
+
+  def initialize
+    @genres = read_genres
+  end
```
[the remark this answers is not in the document]
Comment (Nemwel-Boniface) [reply musicAlbum/genre_controller.rb:10]: Thank you Levy for this suggestion, let me effect it immediately.

**Comment by Nemwel-Boniface** [comment]
Hello, @levy002  I have made the requested changes and now it is working exactly as expected. I will merge the PR.

**Pull request merged**